src/styles/arrows.scss

```diff
@@ -0,0 +1,41 @@
+.arrows__arrow {
```

Comment:
also wrap all styles which are related to arrow in `.arrows { and use nesting inside &__arrow}`

Reply:
just for clarity